src/validation.h

```diff
@@ -435,6 +610,9 @@ bool InvalidateBlock(CValidationState& state, const CChainParams& chainparams, C
 /** Remove invalidity status from a block and its descendants. */
 void ResetBlockFailureFlags(CBlockIndex* pindex) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+/** @returns the most-work chainstate. */
```

Comment:
```suggestion
/** @returns the most-work valid chainstate. */
```

in commit 7794d41a58 refactoring: introduce ChainstateActive()


Could clarify that this the chainstate is fully validated (as opposed to a pow header chain)

Reply:
Ah good point! Thanks.